charmcraft.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+type: "charm"
+bases:
+  - build-on:
+    - name: "ubuntu"
+      channel: "20.04"
+    - name: "ubuntu"
+      channel: "22.04"
+    run-on:
+    - name: "ubuntu"
+      channel: "20.04"
+
+parts:
+  charm:
+    build-packages:
+      - libffi-dev
+      - libssl-dev
+      - rustc
+      - cargo
+    charm-binary-python-packages:
+      - psycopg2-binary
```

Comment:
Upstream recommends against using the binary package: https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary

Reply:
Oops, and we don't even need this package! This came from copy pasting from a different project. I removed it.